updating/understanding-openshift-upgrade-duration.adoc

```diff
@@ -0,0 +1,91 @@
+:_content-type: ASSEMBLY
+[id="understanding-openshift-upgrade-duration"]
+= Understanding openshift upgrade duration
```

Comment:
These docs get rendered into a few products, right?  It's not just OpenShift.  Can we leave the product implicit in the filename and title, and use `{product-title}` in the doc body?

Reply:
Done.